instrumentation/src/java/org/oppia/android/instrumentation/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "oppia_test_application",
+    testonly = True,
+    srcs = [
+        "EndToEndTestGcsResourceModule.kt",
+        "EndToEndTestImageParsingModule.kt",
+        "EndToEndTestNetworkModule.kt",
+        "OppiaTestApplication.kt",
+        "TestApplicationComponent.kt",
+    ],
+    visibility = ["//:oppia_testing_visibility"],
```

Comment:
I think we probably will need a separate package group just for e2e test visibility. Please create one of those (under instrumentation) & use that here & elsewhere for e2e-only dependencies (we really don't want these used elsewhere in tests).

Reply:
Done